docs/source/en/intro/quickstart.md

```diff
@@ -408,6 +408,7 @@ with the above short sections.
 We recommend that developers continue reading other documents:
 - Egg provides a powerful mechanism for extending features. See [Plug-ins](../advanced/plugin.md).
 - As large teams need to follow certain constraints and conventions, in egg, we recommend further creating an upper-level framework that fits your team. See [Frameworks](../advanced/framework.md).
+- Egg is a progressive framework, reuse of code is painless. See [Progressive](../tutorials/progressive.md).
```

Comment:
Egg framework enables developers to write painless unit testing with many plugins and community-powered toolings. The team should give it a try by using Egg unit testing without worrying about setting up the testing tooling but writing the testing logics.